ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallFooter.kt

```diff
@@ -13,18 +13,45 @@ import androidx.compose.ui.Modifier
 /**
  * Composable offering a minified screen Paywall UI configured from the RevenueCat dashboard.
  * You can pass in your own Composables to be displayed in the main content area.
+ * Only compatible with original Paywalls. If using V2 paywalls, this will display a fallback paywall instead.
```

Comment:
I noticed this was capitalized in `PaywallOptions.kt`.
```suggestion
 * Only compatible with original Paywalls. If using V2 Paywalls, this will display a fallback paywall instead.
```